src/modules/pages/components/ExpenditurePage/ExpenditurePage.tsx

```diff
@@ -58,15 +61,31 @@ const validationSchema = yup.object().shape({
   ),
 });
 
-const ExpenditurePage = () => {
+type Props = RouteChildrenProps<{ colonyName: string }>;
+
+const ExpenditurePage = ({ match }: Props) => {
+  if (!match) {
+    throw new Error(
+      `No match found for route in ${displayName} Please check route setup.`,
+    );
+  }
+
+  const { colonyName } = useParams<{
+    colonyName: string;
+  }>();
+
+  const { data: colonyData, loading } = useColonyFromNameQuery({
+    variables: { name: colonyName, address: '' },
+  });
+
   const sidebarRef = useRef<HTMLElement>(null);
   const submit = useCallback((values) => {
     // eslint-disable-next-line no-console
     console.log({ values });
   }, []);
 
   return (
-    <Form
+    <Formik
```

Comment:
Why do you need `Formik` here around the whole block of code? It doesn't seem that it's necessary but maybe I'm missing something... You are using `Form` in `ExpenditureForm` separately and I would just use `Form` around or in `TitleDescription`.

Reply:
I use Formik in all sections because there is a button in the Stages component that triggers validation and sending the form.
```
const Stages = ({ states, activeStateId }: Props) => {
  const { values, handleSubmit, validateForm, resetForm } = useFormikContext<ValuesType>();

  const openDeleteDraftDialog = useDialog(DeleteDraftDialog);
  const openDraftConfirmDialog = useDialog(StakeExpenditureDialog);

  const handleSaveDraft = useCallback(async () => {
    const errors = await validateForm(values);
    const hasErrors = Object.keys(errors)?.length;

    return (
      !hasErrors &&
      openDraftConfirmDialog({
        onClick: () => {
          handleSubmit(values);
        },
      })
    );
  }, [handleSubmit, openDraftConfirmDialog, validateForm, values]);

  const handleDeleteDraft = () =>
    openDeleteDraftDialog({
      onClick: () => {
        resetForm?.();
    });
```

Reply:
ok, thank you.